src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/helper.py

```diff
@@ -352,13 +358,13 @@ def write_output_to_file(output_json: dict, output_file_path: Path) -> None:
             metadata_file_name = output_file_path.with_suffix(".json").name
             metadata_file_path = metadata_dir / metadata_file_name
             # Ensure the metadata directory exists
-            metadata_dir.mkdir(parents=True, exist_ok=True)
+            fs.mkdir(create_parents=True, path=metadata_dir)
```

Comment:
@harini-venkataraman will this behave the same way as the previous code? I hope we won't raise an error if the directory is already present

Reply:
@chandrasekharan-zipstack We dont raise an error if the directory is already present. We log a debug message, and pass. This is handled in the abstracted APIs.